src/api/app/views/webui/project/subprojects.html.haml

```diff
@@ -6,34 +6,46 @@
 
 - if @parentprojects.present?
   %h3 Parent projects of #{@project}
-  %table.compact.stripe.no-footer#parentprojects_table
+  %table.compact.stripe.no-footer{:id => "parentprojects_table", :style => "table-layout:auto; width: 100%;word-wrap:break-word;"}
```

Comment:
In general I would prefer to not use inline styles and using the new ruby hash syntax:

```
id: :parentprojects_table
```